boards/nxp/lpcxpresso55s69/lpcxpresso55s69_lpc55S69_cpu0_ns_defconfig

```diff
@@ -1,21 +1,17 @@
 #
-# Copyright (c) 2019, NXP
+# Copyright 2019,2024 NXP
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
-CONFIG_SOC_SERIES_LPC55XXX=y
-CONFIG_SOC_LPC55S69_CPU0=y
-CONFIG_BOARD_LPCXPRESSO55S69_CPU0=y
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_GPIO=y
 CONFIG_PINCTRL=y
 
-# TFM sets up MPU_NS, can't correctly change this configuration yet
-CONFIG_ARM_MPU=n
+CONFIG_ARM_MPU=y
```

Comment:
Is this change intentional? It looks like we are enabling the ARM MPU for the `ns` target where it previously wasn't

Reply:
Intentional but after a second look should not be in a part of this PR.